integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -40,8 +40,8 @@
  */
 public class Hooks {
 
-    private static final String WEBHOOK_ID = Utils.getSystemProperty("webhook.site.uuid");
-    private static final String WEBHOOK_ID_SECOND = Utils.getSystemProperty("webhook.site.uuid.second");
+    static String WEBHOOK_ID;
```

Comment:
Do we need to declare those ids as properties?

Reply:
Adjusted the code.